src/utils/KeycloakAuth.js

```diff
@@ -0,0 +1,92 @@
+import Keycloak from 'keycloak-js';
```

Comment:
+1 for the refactor. I should have put all KC stuff in it's own file to start with, thanks for doing that :)